Fix bug with current_time microseconds (#17751)
yiisoft/yii2: framework/CHANGELOG.md

2.0.32 under development
------------------------

> - Bug #17744: Fix bug with setting incorrect value for column with `CURRENT_TIMESTAMP(x)` expression for MySQL >= 5.6.4 (bizley)

Reviewer: ```suggestion
- Bug #17744: Fix a bug with setting incorrect `defaultValue` to AR column with `CURRENT_TIMESTAMP(x)` as default expression (MySQL >= 5.6.4 (bizley))
```
